DOCKER/Dockerfile

```diff
@@ -1,80 +0,0 @@
-# Pull base image.
-FROM quay.io/eris/build
-MAINTAINER Eris Industries <support@erisindustries.com>
-
-
-#-----------------------------------------------------------------------------
-# install eris-db
-#
-ENV GO15VENDOREXPERIMENT 1
-
-# install glide for dependency management
-RUN	go install github.com/Masterminds/glide
-
-# set the repo and install erisdb
-ENV REPO $GOPATH/src/github.com/eris-ltd/eris-db
-WORKDIR $REPO
-COPY . $REPO
-RUN glide install
-RUN cd ./cmd/eris-db && go build -o /usr/local/bin/eris-db && cd ../..
```

Comment:
I don't think the final cd ../.. is needed; RUN always executes to whatever WORKDIR is set to